src/site/content/en/blog/payment-and-address-form-best-practices/index.md

```diff
@@ -0,0 +1,782 @@
+
+### Use HTML elements as intended {: #html-elements}
+
+#### Put your form in a &lt;form&gt; {: #html-form}
+
+You might be tempted not to bother wrapping your `<input>`s in a `<form>`, and to handle data 
+  submission purely with JavaScript. 
+
+Don't do it!
+
+An HTML `<form>` gives you access to a powerful set of built-in features across all modern browsers, 
+  and can help make your site accessible to screen readers and other assistive devices. A `<form>` 
+  also makes it simpler to build basic functionality for older browsers with limited JavaScript 
+  support, and to enable form submission even if there's a glitch with your code—and for the 
+  small number of users who actually disable JavaScript.
+
+If you have more than one page component for user input, make sure to put each in its own `<form>` 
+  element. For example, if you have search and sign-up on the same page, put each in its own `<form>`.
+
+#### Use `<label>` to label elements {: #html-label}
+
+To label an `<input>`, `<select>` or `<textarea>`, use a [`<label>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label).
+
+You associate a label with an input by giving the label's `for` attribute the same value as 
+the input's `id`.
+
+```html
+<label for="address-line1">Address line 1</label>
+<input id="address-line1" …>
+```
+Use a single label for a single input: don't try to label multiple inputs with only one label. This 
+works best for browsers, and best for screenreaders. A tap or click on a label moves focus to the input 
+it's associated with, and screenreaders announce label text when the *label* or the label's *input* 
+gets focus.
+
+{% Aside 'caution' %}
+[Placeholders](https://www.smashingmagazine.com/2018/06/placeholder-attribute/) can be handy, but 
+don't use them on their own instead of labels. Once you start entering text in an input, the 
+placeholder is hidden, so it can be easy to forget what the input is for. The same is true if 
+you use the placeholder to show the correct format for values such as dates. This can be especially 
+problematic for users on phones, particularly if they're distracted or feeling stressed! 
+{% endAside %}
+
+#### Make buttons helpful {: #html-button}
+
+Use [`<button>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button)
+for buttons! You can also use `<input type="submit">`, but there's no point in using a div or some 
+other random element acting as a button. Button elements provide accessible behaviour and built-in 
+form submission functionality, and can easily be styled. 
+
+Give each form submit button a value that says what it does. For each step towards checkout, use 
+a descriptive call-to-action that shows progress and makes the next step obvious. For example, label 
+the submit button on your delivery address form **Proceed to Payment** rather than **Continue** 
+or **Save**.
+
+{: #disable-submit}
```

Comment:
Subheading is missing here, I'm not sure what level it should be (3 or 4?) so I'll leave it to you.

Reply:
Okay, upon reading through and seeing a few more anchors without subheading, I suppose that's intentional omission.